nss-day-cohort-47/Class-Website
# updating layout

## Link to Ticket

Updating homepage 

## Description of Proposed Changes
- Updated favicon
- Add header section
- Updated copy content
- Tweaking page layout

## Steps to Test

Just check the layout - still updating design 


```sh
git fetch --all
git checkout lm-homepageDesign

```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* It should not change things structurally, but I did remove the section for class photos. 

## Mentions @coltmay @micaelacolpi @prestonshotts 

Tag users that need to review this code
